eclipsesource/theia-cloud
# Add authentication to the try now page

Moved components into separate files.
Added header that contains login/logout buttons.
User email will be displayed if logged in.
Added hint for why the login is needed.

Contributed on behalf of STMicroelectronics

Additionally Github integration was added to the keycloak.

To test:
- Enable the keycloak
- Point `keycloak.authUrl` to the keycloak (https://keycloak.34.141.62.32.nip.io/) (`/auth` is no longer needed)
- Build and use the try-now-page image

Note:
I think we should discuss, how we want to proceed with email login. It is currently still shown on the keycloak login page, but we do not support any registration process. This might be confusing.
So we should either:
- Remove the email login from the keycloak (requires custom theme)
- or add a registration process (might not be used by a lot of people)
